docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -7,19 +7,19 @@ The main features of the Image Util API include:
 
 - Conversion
 
-  You can [change the color space format](#colorspace) among the [supported formats](#color_format).
+  You can change the [image](#image_colorspace) and [media packet](#mediapacket_colorspace) color space among the [supported formats](#color_format).
 
 - Resizing
 
-  You can [change the image resolution](#resize).
+  You can change the [image](#resize_image) and [media packet](#resize_mediapacket) resolution.
 
 - Rotation
 
-  You can [change the image angle](#rotate) around the X or Y axis.
+  You can change the [image](#rotate_image) and [media_packet](#rotate_mediapacket) angle around the X or Y axis.
 
 - Crop
 
-  You can [remove the outer parts of an image](#crop) or change the aspect ratio.
+  You can remove the outer parts of the [image](#crop_image) and [media packet](#crop_mediapacket).
 
 - Decoding from a file or memory and encoding to a file or memory
```

Comment:
Just  a suggestion:

Can we avouid using sub bullet for the Bitmap formts, Input image formats for decoding, and Output image formats for encoding?

Suggest rephrasing to:
- Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, ARGB8888
- Input image formats for decoding: JPEG, PNG, GIF, BMP
- Output image formats for encoding: JPEG, PNG, GIF, BMP, animated GIF

Reply:
Is the note unnecessary?

Reply:
@jiyong2, sorry for the miss, please remove the periods at the end of the sentences:

Suggest rephrasing to:

Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
Input image formats for decoding: JPEG, PNG, GIF, and BMP
Output image formats for encoding: JPEG, PNG, GIF, BMP, and animated GIF